diff --git a/configure.ac b/configure.ac
index 5dff4c3..f0ba052 100644
--- a/configure.ac
+++ b/configure.ac
@@ -35,14 +35,14 @@ djvulibre_version=$($PKG_CONFIG --modversion ddjvuapi)
 AC_DEFINE_UNQUOTED([DJVULIBRE_VERSION_STRING], ["$djvulibre_version"], [Define to the version of DjVuLibre])
 
 AC_MSG_CHECKING([DjVuLibre fitness])
-for tool in bzz c44 cjb2 csepdjvu djvuextract djvumake djvused djvmcvt
-do
-  if ! test -x "$djvulibre_bin_path/$tool$EXEEXT"
-  then
-    AC_MSG_RESULT([incomplete set of tools])
-    AC_MSG_ERROR([$tool$EXEEXT not found])
-  fi
-done
+dnl for tool in bzz c44 cjb2 csepdjvu djvuextract djvumake djvused djvmcvt
+dnl do
+dnl   if ! test -x "$djvulibre_bin_path/$tool$EXEEXT"
+dnl   then
+dnl     AC_MSG_RESULT([incomplete set of tools])
+dnl     AC_MSG_ERROR([$tool$EXEEXT not found])
+dnl   fi
+dnl done
 AC_MSG_RESULT([ok])
 
 PKG_CHECK_MODULES([POPPLER], [poppler-splash >= 0.11.1])
