diff --git a/cli/src/lib.rs b/cli/src/lib.rs
index 945fe33..463b4cd 100644
--- a/cli/src/lib.rs
+++ b/cli/src/lib.rs
@@ -10,7 +10,7 @@ pub mod test;
 pub mod test_highlight;
 pub mod util;
 pub mod wasm;
-pub mod web_ui;
+// pub mod web_ui;
 
 #[cfg(test)]
 mod tests;
diff --git a/cli/src/main.rs b/cli/src/main.rs
index 8f5fdb4..cad0695 100644
--- a/cli/src/main.rs
+++ b/cli/src/main.rs
@@ -7,7 +7,7 @@ use std::{env, fs, u64};
 use tree_sitter::Language;
 use tree_sitter_cli::{
     config, error, generate, highlight, loader, logger, parse, query, test, test_highlight, wasm,
-    web_ui,
+    // web_ui,
 };
 
 const BUILD_VERSION: &'static str = env!("CARGO_PKG_VERSION");
@@ -290,7 +290,9 @@ fn run() -> error::Result<()> {
         let grammar_path = current_dir.join(matches.value_of("path").unwrap_or(""));
         wasm::compile_language_to_wasm(&grammar_path, matches.is_present("docker"))?;
     } else if matches.subcommand_matches("web-ui").is_some() {
-        web_ui::serve(&current_dir);
+        print!("ERROR: web-ui is not available in the nixpkgs tree-sitter-cli at the moment.");
+        std::process::exit(1);
+        // web_ui::serve(&current_dir);
     } else if matches.subcommand_matches("dump-languages").is_some() {
         loader.find_all_languages(&config.parser_directories)?;
         for (configuration, language_path) in loader.get_all_language_configurations() {
